docs/administration.index.md

```diff
@@ -3,32 +3,31 @@ id: administration.index
 title: Administration Roles
 ---
 
-Partners Users have access to the so called **Administration Area**. The administration area enables partner users to manage their child [meshWorkspace](./meshcloud.workspace.md) accounts within the meshcloud platform.
-Assigning Partner Users can only be done by Partner Admins.
+Admin users have access to the so called **Administration Area**. The administration area enables admin users to manage their [workspace](./meshcloud.workspace.md) accounts within the meshcloud platform.
+Assigning admin users can only be done by user who have "organization admins" rights.
 
 Different groups of people may need access to the different administration area functionality. Therefore, the following administrative roles are currently
 available to users of the administration area:
 
-- **Partner Admin**: It's basically the account with administration rights. Has full access to all functionality and can manage users of the
```

Comment:
d: from this whole section I don't think we should apply the lower case here. This role name are used as enums. So title case should be considered correctly this is like an introduction with the available role name